steam-agreement-ok-leap-20180401.json

```diff
@@ -0,0 +1,24 @@
+{

```

Comment:
Why you need this needle ? Looks identical to steam-agreement-ok but with console in background which should be unrelated.

Reply:
The PR includes two variants, for openSUSE Tumbleweed, no prefix in name, and openSUSE Leap, with "-leap-" in name